src/otpch.h

```diff
@@ -30,7 +30,7 @@
 #include "features.h"
 
 // Canary Lib
-#include "lib/include.hpp"
+#include "../canary-lib/include.hpp"
```

Comment:
@Costallat I wish to somehow move it to "canary-lib/include.hpp" via absolute path using target_link_directories, but I couldn't manage to do it. Ideas?